diff -ruN libcxx-14.0.0rc2.src.orig/CMakeLists.txt libcxx-14.0.0rc2.src/CMakeLists.txt
--- a/libcxx-14.0.0rc2.src.orig/CMakeLists.txt	2022-03-04 17:08:07.363822171 +0100
+++ a/libcxx-14.0.0rc2.src/CMakeLists.txt	2022-03-07 08:29:06.632754832 +0100
@@ -13,6 +13,7 @@
   "${LLVM_COMMON_CMAKE_UTILS}/Modules"
   )
 
+
 set(CMAKE_FOLDER "libc++")
 
 set(LIBCXX_SOURCE_DIR  ${CMAKE_CURRENT_SOURCE_DIR})
@@ -40,6 +41,7 @@
 include(GNUInstallDirs)
 
 if (LIBCXX_STANDALONE_BUILD)
+  find_package(LLVM REQUIRED HINTS "${LLVM_CMAKE_DIR}")
 
   find_package(Python3 COMPONENTS Interpreter)
   if(NOT Python3_Interpreter_FOUND)
